fs-cache/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod cache;
```

Comment:
All of the following comments are just nitpicks, so feel free to ignore them.

I think it’s better to only export the `Cache` struct. That way, when we import the crate, we can use `Cache` directly without needing to do `fs_cache::cache::Cache`. It would be `fs_cache::Cache` instead.


```suggestion
mod cache;

pub use cache::Cache;
```